### ISSUE TITLE
String form Qobjevo cannot be instantiated

### ISSUE THREAD

**Issue by xspronken** [qutip/qutip#1960]
### Bug Description

String form QobjEvos cannot be created.
The error is due to a missing file: `intdtype.h` is missing from `/home/xavier/anaconda3/envs/qutip5-bench/lib/python3.10/site-packages/qutip/core/data/src/`.

### Code to Reproduce the Bug

```shell
import qutip 
n = qutip.num(5)
string_form= qutip.QobjEvo([n, 'cos(t)'])
```

### Code Output

```shell
[1/1] Cythonizing /home/xavier/.qutip/qutip_coeffs_1.1/qtcoeff_d9971e1c0d4b7f53f4bf8a5dbd7cab.pyx
running build_ext
building 'qtcoeff_d9971e1c0d4b7f53f4bf8a5dbd7cab' extension
gcc -pthread -B /home/xavier/anaconda3/envs/qutip5-bench/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /home/xavier/anaconda3/envs/qutip5-bench/include -fPIC -O2 -isystem /home/xavier/anaconda3/envs/qutip5-bench/include -fPIC -I/home/xavier/anaconda3/envs/qutip5-bench/lib/python3.10/site-packages/numpy/core/include -I/home/xavier/anaconda3/envs/qutip5-bench/include/python3.10 -c /home/xavier/.qutip/qutip_coeffs_1.1/qtcoeff_d9971e1c0d4b7f53f4bf8a5dbd7cab.cpp -o build/temp.linux-x86_64-cpython-310/home/xavier/.qutip/qutip_coeffs_1.1/qtcoeff_d9971e1c0d4b7f53f4bf8a5dbd7cab.o -w -O3 -funroll-loops
/home/xavier/.qutip/qutip_coeffs_1.1/qtcoeff_d9971e1c0d4b7f53f4bf8a5dbd7cab.cpp:749:10: fatal error: src/intdtype.h: No such file or directory
  749 | #include "src/intdtype.h"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
error: command '/usr/bin/gcc' failed with exit code 1
```

### Expected Behaviour

I was expecting a string form QobjEvo object to be created.

### Your Environment

```shell
QuTiP Version:      5.0.0.dev0+0a19bf9
Numpy Version:      1.22.4
Scipy Version:      1.8.1
Cython Version:     0.29.30
Matplotlib Version: 3.5.2
Python Version:     3.10.5
Number of CPUs:     8
BLAS Info:          Generic
INTEL MKL Ext:      False
Platform Info:      Linux (x86_64)
```

### Additional Context

Qutip was installed using `pip install git+https://github.com/qutip/qutip@dev.major` in a conda environment.

**Comment by hodgestar** [comment]
The file `intdtype.h` should be created by setup.py. How did you build your version of QuTiP 5? Perhaps something is missing (e.g. maybe `MANIFEST.in` needs to have a line that reads `recursive-include qutip *.h` (or perhaps just directly only the intdtype header file).

**Comment by xspronken** [comment]
I used `pip install git+https://github.com/qutip/qutip@dev.major ` to install it.

**Comment by xspronken** [comment]
I tried creating a new environment, cloning the dev.major branch and running `setup.py install` which didn't work either.
Adding `recursive-include qutip *.h` to `MANIFEST.in` before running the setup solved the problem.

Interestingly it solved it for the `qutip5-bench` environment as well. So I created two new environments to see if the problem would also be solved in those, installing dev.major with pip in one and by cloning and using `setup.py install` without adding `recursive-include qutip *.h` to `MANIFEST.in` in the other. The QobjEvo object was created without any problems in both environments
